### PR TITLE
login page created

### PR DESCRIPTION
**Pull request by aamrapaliwandhre**
**1)Login Page
2)Updated nav**
chrome-capture (23)

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<html>
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FinestStitch</title>
+    <link rel="stylesheet" href="https://distracted-jones-2f14b5.netlify.app/index.css" />
```
Comment (Snigdha17) [inline login.html]: You imported this library in style.css too, you may not need to import it here :)

**Review by Snigdha17 — APPROVED** [review]
Code looks clean, good use of variables and rems. Left a minor comment, but good to merge

File: login.html (hunk 1 above)
Comment (aamrapaliwandhre) [reply login.html]: okay..removing it

File: login.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    color: var(--white); 
+}
+.dropdown-content {
+    position: absolute;
+    background-color: var(--blueviolet);
+    border-radius: 0.2rem;
+    right: 0.03rem;
+    top: 3.2rem;
+    padding: 0.5rem;
+    padding-left: 0.5rem;
```
Comment (SanjayJatti) [inline login.css:18]: No need of padding-left as you already given padding: 0.5rem

File: login.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    display: none;
+    text-align: center;
+}
+.dropdown-content .a{
+    color: black;
```
Comment (SanjayJatti) [inline login.css:23]: You should use variable for black color also as it would be more consistent.

File: login.css [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    display: block;
+}
+.dropdown-content .a:hover {
+    background-color: var(--light-violet);
+}
+.dropdown:hover .dropdown-content {
+    display: block;
+}
+.dropdown:hover .dropbtn {
+    background-color: grey;
```
Comment (SanjayJatti) [inline login.css]: Use color variable for gray.

**Review by SanjayJatti — APPROVED** [review]

File: login.css (hunk 3 above)
Comment (aamrapaliwandhre) [reply login.css:23]: i forget to use variable in hurry, changing it.

**Pull request merged**